nomos-core/chain-defs/src/sdp/mod.rs

```diff
@@ -213,6 +220,13 @@ impl<Metadata> SdpMessage<Metadata> {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum FinalizedDeclarationState {
```

Comment:
could you elaborate on what this is for?

Reply:
```
pub enum FinalizedDeclarationState {
```

> could you elaborate on what this is for?

At the moment this definition is used in Membership service, where we are supposed to receive the state changes of declarations that are not withdrawn or expired yet.